apps/gallery/js/thumbnail_item.js

```diff
@@ -38,8 +38,7 @@ function ThumbnailItem(fileData) {
 ThumbnailItem.formatter = new navigator.mozL10n.DateTimeFormat();
 
 ThumbnailItem.prototype.localize = function() {
-  var date = new Date(this.data.date);
-  var imageCreatedAt = navigator.mozL10n.get('image-created-at');
-  var label = ThumbnailItem.formatter.localeFormat(date, imageCreatedAt);
+  var label = MediaUtils.shortDescription(
```

Comment:
You mean MediaUtils.shortDescription? I named it so because in a future patch, we will perhaps have longer descriptions for videos/images in fullscreen. The fact that we set it via aria-label seems less relevant (it could just as easily be set on the alt or title attribute). I don't mind changing it though, if you think that would be more descriptive.